src/test/java/org/influxdb/InfluxDBTest.java

```diff
@@ -100,6 +100,37 @@ public void testQuery() {
 	}
 
 	/**
+	 * Test multiple queries with an HTTP GET request return results having the same size of queries.
+	 */
+	@Test
+	public void testMultipleQuery() {
+		String dbName = "unittest_" + System.currentTimeMillis();
+		try {
+			this.influxDB.createDatabase(dbName);
+			QueryResult result = this.influxDB.query(new Query("SELECT * FROM not_exists;SELECT * FROM not_exists", dbName));
+			Assert.assertEquals(2, result.getResults().size());
+		} finally {
+			this.influxDB.deleteDatabase(dbName);
+		}
+
+	}
+
+	/**
+	 * Test multiple queries with an HTTP POST request return results having the same size of queries.
+	 */
+	@Test
+	public void testPostQuery() {
+		String dbName = "unittest_" + System.currentTimeMillis();
+		try {
+			this.influxDB.createDatabase(dbName);
```

Comment:
it is one of best practices instead of if it is necessary for this case. So it is up to you. just one advice for best practice.